spec/index.html

```diff
@@ -1155,6 +1593,285 @@ <h3>Examples</h3>
+          The <a data-cite="INFRA#map-entry">map entry</a> for `_:b0` in the <a>blank node to quads map</a> results in the
+          following quads:</p>
+
+        <pre id="ex-nd-shared-hashes-b0-quads" data-transform="updateExample">
+          <!--
+            :p :q _:b0 .
+            _:b0 :p _:b2 .
+          -->
+        </pre>
+
+        <p><a href="#hndq-3">Step 3</a> iterates over each of these quads
+          to find <a>blank node</a> <var>components</var>
+          to populate <var>H<sub>n</sub></var>
+          with the result of running the
+          <a href="#hash-related-blank-node">Hash Related Blank Node algorithm</a>
+          using the <var>position</var> of that <var>component</var> within the quad.</p>
+
+        <table id="ex-nd-shared-table-related-hash-b0"
+               class="hash-related numbered">
+          <caption>Hash for `_:b2`  related to `_:b0`</caption>
+          <thead><tr>
+            <th><var>related</var></th>
+            <th><var>quad</var></th>
+            <th><var>position</var></th>
+            <th><var>hash</var></th>
+          </tr></thead>
+          <tbody>
+            <tr>
+              <td>`_:b2`</td>
+              <td>`_:b0 :p _:b2 .`</td>
+              <td>`o`</td>
+              <td>
+                <!-- canonical identifer _:b2 is _:c14n0 -->
+                <!-- hash "o<http://example.com/#p>_:c14n0"-->
+                <code>29cf7e22790bc2ed395b81b3933e5329fc7b25390486085cac31ce7252ca60fa</code></td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p>which results in the following <var>H<sub>n</sub></var>:</p>
+
+        <table id="ex-nd-shared-table-bn-list-b0"
+               class="hash-related numbered">
+          <caption>Blank node list for hash</caption>
+          <thead><tr>
+            <th><var>related hash</var></th>
+            <th><var>blank node list</var></th>
+          </tr></thead>
+          <tbody>
+            <tr>
+              <td><code>29cf7e22790bc2ed395b81b3933e5329fc7b25390486085cac31ce7252ca60fa</code></td>
+              <td>`[ _:b2 ]`</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p><a href="#hndq-5">Step 5</a> iterates over each <var>related hash</var>
+          which is a key in <var>H<sub>n</sub></var>,
+          which can have one or more related blank nodes,
+          which determines the shortest <a>gossip path</a> between
+          the two nodes (`_:b0` and `_:b2`).
+          In this case, the hash maps to just `_:b2`,
+          for which a canonical identifier has already been chosen (`_:c14n0`),
+          so there is a single permutation
+          resulting in one candiate path.
+          The resulting <var>chosen path</var> is `_:c14n0`.</p>
+
+        <p>The string <var>data to hash</var> is composed
+          of the single <var>related hash</var> and the <var>chosen path</var>:
+          `00e6a7a00375f5072ce6a589d4c7d9fa8833c9f57aaa4f1cb1c84d240e716e3a_:c14n0`,
```

Comment:
Is this correct? The _related hash_, as is also on the table above, is `29cf...60fa` and not this `00e6...6e36`. Ie, the _data to hash_ value should be `29cf...60fa_c:14n0` which, at least in my run, yields:

`fbc300de5afafd97a4b9ee1e72b57754dcdcb7ebb724789ac6a94a5b82a48d30`

Do I miss yet another hashing somewhere?

(I did not check the values for the second run of the cycle.)

Reply:
I re-generated all the outputs from the log output from my implementation.

Reply:
And indeed, your numbers match my implementation's !